evergreen-wellness/evergreen-wellness.php

```diff
@@ -83,3 +83,51 @@ function set_default_admin_color($user_id) {
 	add_action('user_register', 'set_default_admin_color');
 }
 
+/**
+ * Description: Production ID Admin sort and search
+ * Purpose: Add custom sort column for Production ID to POST and VIDEO content types
+ * Notes: Funtion relies on ACF plugin to work properly, field name must be "production_id"
+ * @author  etwilbeck
+ * @since  1.0.0
+ */
+add_filter('manage_posts_columns', 'egw_columns_head');
```

Comment:
Well I had originally thought I would have to use `manage_edit-videos_posts_columns` and `manage_edit-post_posts_columns` and have TWO filters instead of one, because it had to be on BOTH video and regular posts. But then I saw an example using `manage_posts_columns` which said that it did ALL "post" types, even your custom ones, so I knew that was the answer!